test/Engine/ProtoTest/ProtoAST/RoundTripTests.cs

```diff
@@ -295,19 +295,19 @@ public void TestAstToCode()
 
             List<string> statements = new List<string>
             {
-                "x = {};",
+                "x = [];",
                 @"x = [1, true, false, 1.234, 41, null, ""hello world"", 'x', foo(a<1>, b<2>)[3]][0]<1>;",
                 "x = (a..b..c)<1>..(m<1>..n<2>..~1)<2>..(i..j..#k<1>)<3>;",
                 "x = (a > b ? foo(1, 2)<1> : bar(2, 3)<2>) ? ding(1, 2)<1> : dong(1, 2)<2>;", 
                 "x = (a == b) ? ((a >= b) ? (a && b) : (a || b)) : (a < b);",
                 "x = foo(a<1>, b<2>, c<3>)[0]<1>;",
-                "x = Point.ByCoordinates({1,2,3}, {4,5,6}<1>, {7, 8, 9}<2>)<0>;",
+                "x = Point.ByCoordinates([1,2,3], [4,5,6]<1>, [7, 8, 9]<2>)<0>;",
                 "x = this.foo(1, 2);",
                 "x = this.X;",
-                "return = {1, 2, 3};",
+                "return = [1, 2, 3];",
                 "return = Math.PI;",
                 "return = Math.Cos(1.2);", 
-                "x[0][1] = {};",
+                "x[0][1] = [];",
```

Comment:
I'm surprised this wasn't an error I encountered before... Did you just revive this test?

Reply:
Oh I just manually updated it even though it's marked failure. I think your script skipped those tests that were marked as "failure"?